module/platform/darwin/HardwareIO/tests/ConversionTest.cpp

```diff
@@ -146,7 +147,7 @@ TEST_CASE("Testing the hardware position conversions to radians", "[hardware][co
                                                            {3073, M_PI_2},
                                                            {4095, M_PI}};
 
-        for (size_t i = 0; i < 20; ++i) {
+        for (ssize_t i = 0; i < 20; ++i) {
```

Comment:
I would just use int here, yes `ssize_t` is technically not a bad choice, but having choices makes it harder on your brain. If you can use int, use int :P